src/ducks/subject.js

```diff
@@ -241,9 +261,12 @@ const fetchSubject = (initialFetch = false) => {
         });
     };
 
-    if (savedClassificationPrompt) {
+    if (checkEmergencySave()) {  //If not, check if there's an emergency save.
```

Comment:
How do we know the emergency save belongs to the user that is currently classifying? It is possible to create an emergency classification with a logged in user, sign out on the main page, then enter the classifier and continue that classification as a logged out user. I think the emergency save local storage item should be identified by the current user as the saved classification prompt currently is.